platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/reconnect/ReconnectProtocolTests.java

```diff
@@ -123,6 +129,9 @@ public String toString() {
     void setup() {
         activeStatusGetter = mock(PlatformStatusGetter.class);
         when(activeStatusGetter.getCurrentStatus()).thenReturn(PlatformStatus.ACTIVE);
+
+        reconnectController = mock(ReconnectController.class);
+        when(reconnectController.blockLearnerPermit()).thenReturn(true);
```

Comment:
Should there be a test for "`blockLearnerPermit` returning false" scenario?

Reply:
Its an almost impossible scenario, but I added a test for it. Please re-approve